filetracker/servers/base.py

```diff
@@ -23,8 +23,15 @@ class Server(object):
 
     def __call__(self, environ, start_response):
         try:
-            return getattr(self, 'handle_' + environ['REQUEST_METHOD']) \
-                    (environ, start_response)
+            if environ['REQUEST_METHOD'] == 'HEAD':
+                environ['REQUEST_METHOD'] = 'GET'
+                _body_iter = self.__call__(environ, start_response)
```

Comment:
I have a small concern that for example when discarding the `_fileobj_iterator` here, we're not closing the file (because the iterator only closes it when all has been read).
However it's most likely a non-issue, because the file was opened in read only mode, so we don't need to close it urgently and the GC should close it when the iterator is freed.